src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -1470,9 +1522,36 @@ public void ProcessFileChange(string projectSystemFilePath, string workspaceFile
 
                 ClearAndZeroCapacity(_documentsRemovedInBatch);
 
+                // Update project's order of documents.
+                solution = UpdateProjectDocumentsOrderForBatch(solution);
+
                 return solution;
             }
 
+            internal Solution UpdateProjectDocumentsOrderForBatch(Solution solution)
```

Comment:
Inline this, since it's only being used in one place? (I think it'll fit with the existing pattern better)

Reply:
Ok, will do.